inst/pages/IntegratedLearner.qmd

```diff
@@ -0,0 +1,440 @@
+# Multi-omics Integration {#sec-multi-omics-integration}
+
+```{r setup, echo=FALSE, results="asis"}
+library(rebook)
+chapterPreamble()
+```
+
+## Multi-omics Integration
+
+In multiview data analysis, there are two main types of approaches: early fusion and late fusion. \underline{Early fusion} combines all datasets into a single representation, which then serves as input for a supervised learning model. In contrast, \underline{late fusion} builds individual models for each data view and combines their predictions using a second-level model as the final predictor. However, these traditional paradigms treat the data views in isolation and do not allow for interactions or dependencies between them. A more advanced method, called cooperative learning (Ding et al., 2022, \emph{PNAS}$^1$), which is also known as \underline{intermediate fusion}, combines the best of both worlds by encouraging predictions from different data views to align through an agreement parameter ($\rho$).
+
+## Multi-omics Prediction and Classification of Binary IBD Disease Status
+
+In this chapter, we showcase examples of various integration paradigms (early, late, and intermediate fusion) using the R packages `multiview`$^1$ and   `SuperLearner`$^2$. We make use of the publicly available source code from the multi-omics integrated framework (`IntegratedLearner`) proposed by Mallick et al. (2024)$^3$.
+
+## Input data
+We use the publicly available Inflammatory Bowel Diseases (IBD) data from the curatedMetagenomicData package (Lloyd-Price et al., 2019)$^4$, where we aim to predict IBD disease status based on both taxonomic (species abundances) and functional (pathway abundances) profiles.
+
+```{r message=FALSE, warning=FALSE}
+
+#####################
+# Load example data #
+#####################
+
+library(tidyverse)
+library(curatedMetagenomicData)
+library(curatedMetagenomicAnalyses)
```

Comment:
Hello!

I started to review this, but ended up to issue. From where this "curatedMetagenomicAnalyses" package comes from? It seems that it is not in CRAN or Biconductor. Can it be avoided somehow?